ambry-api/src/main/java/com.github.ambry/config/ServerConfig.java

```diff
@@ -61,6 +61,13 @@
   @Default("false")
   public final boolean serverEnableStoreDataPrefetch;
 
+  /**
+   * Implementation class for StoreKeyConverterFactory
+   */
+  @Config("server.store.key.converter.factory")
+  @Default("com.github.ambry.server.StoreKeyConverterFactoryImpl")
+  public final String serverStoreKeyConverterFactoryClass;
```

Comment:
the variable has a `Class` at the end that you probably want to remove.

Reply:
fixed